solutions/observability/logs/stream-any-log-file-using-edot-collector.md

```diff
@@ -0,0 +1,374 @@
+        host.id:
+          enabled: false
+        host.arch:
+          enabled: true
+        host.ip:
+          enabled: true
+        host.mac:
+          enabled: true
+        host.cpu.vendor.id:
+          enabled: true
+        host.cpu.family:
+          enabled: true
+        host.cpu.model.id:
+          enabled: true
+        host.cpu.model.name:
+          enabled: true
+        host.cpu.stepping:
+          enabled: true
+        host.cpu.cache.l2.size:
+          enabled: true
+        os.description:
+          enabled: true
+        os.type:
+          enabled: true
+
+exporters:
+  # Exporter to print the first 5 logs/metrics and then every 1000th
+  debug:
+    verbosity: detailed
+    sampling_initial: 5
+    sampling_thereafter: 1000
+
+  # Exporter to send logs and metrics to Elasticsearch Managed OTLP Input
+  otlp/ingest:
+    endpoint: ${env:ELASTIC_OTLP_ENDPOINT}
+    headers:
+      Authorization: ApiKey ${env:ELASTIC_API_KEY}
+
+service:
+  extensions: [file_storage]
+  pipelines:
+    logs/platformlogs:
+      receivers: [filelog/platformlogs]
+      processors: [resourcedetection]
+      exporters: [debug, otlp/ingest]
+```
+:::
+:::
+::::
+
+Key configuration elements:
+
+* `receivers.filelog/platformlogs.include`: Specifies the path to your log files. You can use patterns like `/var/log/*.log`.
+* `processors.resourcedetection`: Automatically detects and adds host system information to your logs.
+* `extensions.file_storage`: Provides persistent storage for the collector's state.
+* `exporters`: Configures how data is sent to Elasticsearch (Elastic Stack) or OTLP endpoint (Serverless).
+:::::
+
+:::::{step} Run the EDOT Collector
+
+Run the following command to run the EDOT Collector:
+
+::::{tab-set}
+
+:::{tab-item} Linux and macOS
+
+```bash
+sudo ./otelcol --config otel.yml
+```
+:::
+
+:::{tab-item} Windows
+
+```powershell
+.\elastic-agent.exe otel --config otel.yml
+```
+:::
+
+::::
+
+:::{note}
+The Collector opens ports `4317` and `4318` to receive application data from locally running OTel SDKs without authentication. This allows the SDKs to send data without any further configuration needed as they use this endpoint by default.
+:::
+:::::
+::::::
+
+## Troubleshoot your EDOT Collector configuration [logs-stream-edot-troubleshooting]
+
+If you're not seeing your log files in the UI, verify the following:
+
+* The path to your logs file under `include` is correct.
+* Your API key is properly set in the environment variables.
+* The OTLP endpoint URL is correct and accessible.
+* The Collector is running without errors (check the console output).
+
+If you're still running into issues, see [EDOT Collector troubleshooting](/troubleshoot/ingest/opentelemetry/edot-collector/index.md) and [Configure EDOT Collector](opentelemetry://reference/edot-collector/config/index.md).
+
+## Next steps [logs-stream-edot-next-steps]
+
+After you have your EDOT Collector configured and are streaming log data to {{es}}:
```

Comment:
Maybe also link to the [Explore logs in Discover](https://www.elastic.co/docs/solutions/observability/logs/discover-logs) docs?

Reply:
Added!